aqua/jumpcut/Portfile

```
@@ -1,33 +1,53 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           xcode 1.0
 
 name                jumpcut
-version             0.63
 categories          aqua
-maintainers         nomaintainer
+maintainers         {judaew @judaew} openmaintainer
 license             MIT
-supported_archs     i386 ppc
 
 description         Jumpcut: Minimalist Clipboard Buffering for OS X
-
 long_description    Jumpcut is an application that provides “clipboard \
                     buffering” — that is, access to text that you’ve cut or \
                     copied, even if you’ve subsequently cut or copied \
                     something else. The goal of Jumpcut’s interface is to \
                     provide quick, natural, intuitive access to your \
                     clipboard’s history.
-
-homepage            http://jumpcut.sourceforge.net/
-master_sites        sourceforge:project/jumpcut/jumpcut-source/${version}/
-
-distname            ${name}_${version}-src
-extract.suffix      .tgz
-
-checksums           md5     8ea757ff5723c397be557618592e49bc \
-                    rmd160  ddf4f7fee02d84de62cec037a286d5f9c8a704f1 \
-                    sha256  48a0d44673138a97a2e734cd991bb065437ec66e1653a44d0e2679aa1087c9d9 \
-                    size    486246
-
-patchfiles          patch-AppController.m.diff
+homepage            https://snark.github.io/jumpcut/
+
+if {${os.major} < 14 } {
+    version             0.63
+    revision            0
```

It looks like you're changing this port from build-from-source to install-binary. Since the pre-built binary is probably not identical to the one we would have built from source, probably a good idea to increase the revision here so that everyone has the same version.
```suggestion
    revision            1
```

I assume the installed binary is universal. If so, ensure that the universal variant is enabled, or better yet, strip out the architectures that were not requested if the user installs without the universal variant.